daktari/config.py

```diff
@@ -17,14 +21,55 @@ class Config:
     min_version: Optional[str]
     title: Optional[str]
     checks: List[Check]
+    ignored_checks: List[Check] = field(default_factory=list)
+    printLocalFileMessage: bool = False
 
 
 version_regex = re.compile('daktari_version.*"([0-9.]+)"')
+LOCAL_CONFIG_PATH = "daktari-local.yml"
 
 
 def read_config(config_path: Path) -> Optional[Config]:
     raw_config = config_path.read_text()
-    return parse_raw_config(config_path, raw_config)
+    config = parse_raw_config(config_path, raw_config)
+    if config is None:
+        return config
+
+    return apply_local_config(config)
```

Comment:
I kinda hate Python's ternaries so I tend to avoid them, but that's just personal preference. The positioning is just... odd, like in other languages it's normally:

```
foo = bar ? value_if_true : value_if_false
foo = if (bar) value_if_true else value_if_false
```

But python's the ordering is all messed up, with the conditional sandwiched in the middle. Which makes it harder to read, IMO. Probably is the more pythonic way to use it, though :sweat_smile:

Reply:
Aha I do agree with you, it should be `if (bar) value_if_true else value_if_false`.

I'm happy for you to leave as is 😄